src/terminal/terminal.rs

```diff
@@ -71,6 +71,8 @@ where
     /// Last known position of the cursor. Used to find the new area when the viewport is inlined
     /// and the terminal resized.
     last_known_cursor_pos: (u16, u16),
+    /// Number of frames rendered up until current time.
+    frame_count: u64,
```

Comment:
Should we use `u64`, `u32` or `usize` here? 

I think it is should be `usize` so it is more ergonomic to do `f.count() / vector.len()`

Reply:
Yep let's use the most convenient type imo

Reply:
I changed it to `usize` since that is still plenty of time even at 60 fps.

![image](https://github.com/ratatui-org/ratatui/assets/1813121/c0eb045b-a5dc-4991-95ce-80763196217e)

Reply:
Just thought of that, this is going to be u16 on 16bits devices (~18mins). Is it a problem?
Probably not, it won't crash, just go back to zero.

Reply:
I think it is not a problem since I expect people using it will write code assuming it rolls over. It is also documented to roll over. I think it is worth using `usize` just for the ergonomic benefits since I expect most commonly they are going to doing modulo division.

Reply:
That's what I thought, seems reasonable.